### ISSUE TITLE
Geoserver war includes two (different) versions of the netcdf library

### ISSUE THREAD

**Issue by julian1** [aodn/geoserver-build#203]
```
$ unzip -l  geoserver-1.0.0-imos.war  | grep netcdf
 25453334  2016-03-15 11:00   WEB-INF/lib/netcdfAll-4.6.4.jar
    18360  2016-09-30 11:25   WEB-INF/lib/netcdf-iterator-1.0.0-SNAPSHOT.jar
    59295  2016-10-19 10:44   WEB-INF/lib/netcdf4-4.6.6.jar

```

**Comment by jonescc** [comment]
The netcdf4-4.6.6 dependency just includes netcdf4 writing capability (the hdf iosp).  I don't think we are actually using this at the moment - geoserver-build certainly doesn't.

**Comment by jonescc** [comment]
The hdf iosp is also included in netcdfAll dependency - the netcdf4-4.6.6 dependency removed in the convert-to-java branch.
